lib/asciidoctor/path_resolver.rb

```diff
@@ -255,6 +258,11 @@ def partition_path path, web_path = false
     # shift twice for a UNC path
     if root == DOUBLE_SLASH
       path_segments = path_segments[2..-1]
+    # shift twice for a file:/// path and adjust root
```

Comment:
To remind myself I may want to implement it in the future. I haven't decided yet whether the code is correct how it is or whether this is needed. Need more time to think about it.